jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/test/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImplTest.java

```diff
@@ -55,4 +91,60 @@ private void assertMethodResult(final Method method, final Class<?> returnType,
         }
     }
 
+    @Test
+    public void testGetProcesses(){
```

Comment:
@bsremac I think it would be great if we extend the coverage to all methods in this class. I'm not expecting much to change in here.